File: JavaScript/프로그래머스/1/12901. 2016년/2016년.js
function solution(a, b) {
    const dates = {
        1: 31,
        2: 29,
        3: 31,
        4: 30,
        5: 31, 
        6: 30,
        7: 31, 
        8: 31,
        9: 30,
        10: 31,
        11: 30,
        12: 31,
    }
    const days = ["FRI","SAT","SUN","MON","TUE","WED","THU"]
    let totalDays = 0
    for (month in dates) {
        if (month === String(a)) {
            totalDays += b
            totalDays -= 1
            break
        }
        totalDays += dates[month]
    }
    console.log(totalDays)
    return days[totalDays%7]
}

File: Python3/프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    dates = {
        1: 31,
        2: 29,
        3: 31,
        4: 30,
        5: 31, 
        6: 30,
        7: 31, 
        8: 31,
        9: 30,
        10: 31,
        11: 30,
        12: 31,
    }
    days = ["FRI","SAT","SUN","MON","TUE","WED","THU"]
    total_days = 0
    for month in dates:
        if month == a:
            total_days += (b - 1)
            break
        total_days += dates[month]
    return days[total_days%7]
            